src/extensionInstall.mts

```diff
@@ -0,0 +1,71 @@
+import electron, { ProgressReporter } from '@vscode/test-electron';
+import { spawnSync } from 'node:child_process';
+import { createRequire } from 'node:module';
+import path from 'node:path';
+
+const require = createRequire(import.meta.url);
```

Comment:
Reading the JSON using fs operations is probably preferable here, since we don't really need/want the caching the require gives us

Reply:
I figured since it was a configuration file and that we basically would lose the state on every invocation that the caching would be acceptable, but I will adjust for fs/json

Reply:
Done 965732a8128021ca375de67369a2180b36aa75ae